.nycrc

```diff
@@ -5,10 +5,9 @@
       "examples/*/dist"
     ],
     "exclude": [
-      "packages/*/dist/test/",
       "packages/*/dist/__tests__/",
       "packages/cli/test/",
-      "examples/*/dist/test/",
+      "examples/*/dist/__tests__/",
```

Comment:
I should have made this change in the previous pull request. I think the drop in code coverage is caused by the fact that before, we were including test files in example projects in lines-to-be-covered. Test files usually have 100% coverage, which is skewing the coverage value upwards.

> **coverage/coveralls ** — Coverage decreased (-0.8%) to 90.429%